### PR TITLE
nkushnir selenium

### PR DESCRIPTION
**Pull request by 51stRoR**
Test should work in following way - create Punch In/Out note and compare it's messages and in/out creation datetime with latest record in Records table
test will fail if try to continue create unfinished punch instead of new (happens when previous user left his punch unfinished)

src/main/java/selenium/Selenium.java - raw base Selenium class with base logic and custom checks in main method
src/test/java/SeleniumTest.java - Selenium class logic rewritten in simple TestNG test

### CODE REVIEW

File: src/main/java/selenium/Selenium.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Selenium {
+    static final String username = "TestUser01";
```
Comment (AqaCoreSchool) [inline src/main/java/selenium/Selenium.java]: naming convention says that constants names should be all caps with underscore between words 
private static final String USER_NAME

File: src/test/java/SeleniumTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        WebElement latestRecord = tableRows.get(tableRows.size()-1);
+        List<WebElement> latestRecordBody = latestRecord.findElements(By.xpath("./td"));
+        String recordStartDateTime = latestRecordBody.get(1).getText();
+        String recordPunchInNote = latestRecordBody.get(2).getText();
+        String recordEndDateTime = latestRecordBody.get(3).getText();
+        String recordPunchOutNote = latestRecordBody.get(4).getText();
+        Assert.assertEquals(recordStartDateTime, punchInDateTime, "record Punch In datetime is different then actual Punch In datetime!");
+        Assert.assertEquals(recordPunchInNote, startMessage, "record Punch In note is different then actual Punch In note!");
+        Assert.assertEquals(recordEndDateTime, punchOutDateTime, "record Punch Out datetime is different then actual Punch Out datetime!");
+        Assert.assertEquals(recordPunchOutNote, endMessage, "record Punch Out datetime is different then actual Punch Out datetime!");
```
Comment (AqaCoreSchool) [inline src/test/java/SeleniumTest.java]: this assertion fails

File: src/main/java/selenium/Selenium.java (hunk 1 above)
Comment (51stRoR) [reply src/main/java/selenium/Selenium.java]: fixed

File: src/test/java/SeleniumTest.java (hunk 2 above)
Comment (51stRoR) [reply src/test/java/SeleniumTest.java]: can you please explain more on this?(please check my comments in Telegram)

**Pull request merged**